### PR TITLE
[WIP] Rococo - add new pallets

### PR DESCRIPTION
**Pull request by NukeManDan**
In a transition to a more easy to use for users and stewards of Rococo, this PR adds a few pallets:
- Scheduler & Pre-Image
  - Primarily to schedule Runtime Upgrades and let Parachains know the block this will be included and a headsup to any (breaking) changes that may require their action.
- Identity
  - To enable association of real world ID to accounts if needed to help ID _key_ entities :wink:.

Related #5052  && #4872

### Before merge
- [ ] Resolve all `TODO` blocks
- [x] Add Recovery pallet
- [ ] Benchmarks

## Outstanding Items
- https://github.com/paritytech/substrate/issues/11093 -- Recovery cannot benchmark weights.

### CODE REVIEW

File: runtime/rococo/Cargo.toml [hunk 1]
```diff
@@ -170,8 +176,10 @@ runtime-benchmarks = [
 	"pallet-collective/runtime-benchmarks",
 	"pallet-grandpa/runtime-benchmarks",
 	"pallet-im-online/runtime-benchmarks",
+	"pallet-identity/runtime-benchmarks",
 	"pallet-indices/runtime-benchmarks",
 	"pallet-proxy/runtime-benchmarks",
```
Comment (shawntabrizi) [inline runtime/rococo/Cargo.toml:181]: missing preimage

File: runtime/rococo/src/lib.rs [hunk 2]
```diff
@@ -242,10 +242,15 @@ construct_runtime! {
...
 
+		// Utilities
 		Utility: pallet_utility = 90,
 		Proxy: pallet_proxy = 91,
 		Multisig: pallet_multisig,
-
+		// TODO fix the pallet number/order?
+		Scheduler: pallet_scheduler,
+		Preimage: pallet_preimage,
+		Identity: pallet_identity,
```
Comment (shawntabrizi) [inline runtime/rococo/src/lib.rs:252]: yes, put numbers which are not used. So 92, 93, 94, and 95

**Comment by NukeManDan** [comment]
Going to be covered by a different PR, updating to match kusama as much as feasible line by line, plus minor confined tweaks as required for rococo specific logic

**Pull request closed without merging**